gammapy/maps/wcs/geom.py

```diff
@@ -138,6 +138,11 @@ def data_shape_axes(self):
         """Shape of data of the non-spatial axes and unit spatial axes."""
         return self.axes.shape[::-1] + (1, 1)
 
+    @property
+    def data_shape_image(self):
```

Comment:
Maybe this would be better if private.

Reply:
this is similar to `data_shape_axes` which is public